integrationSystem/notification_sender.bal

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/config;
+import ballerina/log;
+import wso2/gsheets;
+import wso2/gmail;
+
+documentation{A valid access token with gmail and google sheets access.}
+string accessToken = config:getAsString("ACCESS_TOKEN");
+
+documentation{The client ID for your application.}
+string clientId = config:getAsString("CLIENT_ID");
+
+documentation{The client secret for your application.}
+string clientSecret = config:getAsString("CLIENT_SECRET");
+
+documentation{A valid refreshToken with gmail and google sheets access.}
+string refreshToken = config:getAsString("REFRESH_TOKEN");
+
+documentation{Spreadsheet id of the reference google sheet.}
+string spreadsheetId = config:getAsString("SPREADSHEET_ID");
+
+documentation{Sheet name of the reference googlle sheet.}
+string sheetName = config:getAsString("SHEET_NAME");
+
+documentation{Sender email address.}
+string senderEmail = config:getAsString("SENDER");
+
+documentation{The user's email address.}
+string userId = config:getAsString("USER_ID");
+
+documentation{
+    Google Sheets client endpoint declaration with http client configurations.
+}
+endpoint gsheets:Client spreadsheetClient {
+    clientConfig:{
+        auth:{
+            accessToken:accessToken,
+            refreshToken:refreshToken,
+            clientId:clientId,
+            clientSecret:clientSecret
+        }
+    }
+};
+
+documentation{
+    GMail client endpoint declaration with oAuth2 client configurations.
+}
+endpoint gmail:Client gmailClient {
+    clientConfig:{
+        auth:{
+            accessToken:accessToken,
```

Comment:
Need to have **scheme = "oauth"** in the auth configurations

Reply:
scheme and url are added in the connector itself.